library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionHeadersAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package com.chuckerteam.chucker.internal.ui.transaction
+
+import android.text.SpannableStringBuilder
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.chuckerteam.chucker.databinding.ChuckerTransactionItemHeadersBinding
+import com.chuckerteam.chucker.internal.data.entity.HttpHeader
+import com.chuckerteam.chucker.internal.support.applyBoldSpan
+
+internal class TransactionHeadersAdapter : RecyclerView.Adapter<TransactionHeadersAdapter.HeaderViewHolder>() {
+
+    private val headers = arrayListOf<HttpHeader>()
```

Comment:
Generaly speaking, I don't get why we should use `arrayListOf` instead of `mutableListOf`.